packages/babel-preset-react/src/index.js

```diff
@@ -9,32 +9,30 @@ import transformReactPure from "@babel/plugin-transform-react-pure-annotations";
 export default declare((api, opts) => {
   api.assertVersion(7);
 
-  let { pragma, pragmaFrag } = opts;
-
   const {
     pure,
+    pragma,
+    pragmaFrag,
     throwIfNamespace = true,
     useSpread,
-    runtime = "classic",
+    runtime = "automatic",
     importSource,
   } = opts;
 
-  // TODO: (Babel 8) Remove setting these defaults
-  if (runtime === "classic") {
-    pragma = pragma || "React.createElement";
-    pragmaFrag = pragmaFrag || "React.Fragment";
-  }
-
-  // TODO: (Babel 8) Don't cast these options but validate it
-  const development = !!opts.development;
-  const useBuiltIns = !!opts.useBuiltIns;
+  const { development = false, useBuiltIns = false } = opts;
 
   if (typeof development !== "boolean") {
     throw new Error(
       "@babel/preset-react 'development' option must be a boolean.",
     );
   }
 
+  if (typeof useBuiltIns !== "boolean") {
```

Comment:
`useBuiltIns` will be removed in Babel 8, I suggest we leave it as-is and focus on default value of `"runtime"` only.

Reply:
Done.